### PR TITLE
Issue deleting

### PR DESCRIPTION
**Pull request by ActiveChooN**
### Motivation and context

Added issue deleting. External request.

### How has this been tested?

Manually.

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
~~- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly~~
~~- [ ] I have added tests to cover my changes~~
~~- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: cvat-ui/src/reducers/interfaces.ts [hunk 1]
```diff
@@ -678,6 +679,8 @@ export interface ReviewState {
     activeReview: any | null;
     newIssuePosition: number[] | null;
     issuesHidden: boolean;
+    issuesResolvedHidden: boolean;
+    removeIssueDialogShown: boolean;
```
Comment (Marishka17) [inline cvat-ui/src/reducers/interfaces.ts:683]: Is it used anywhere?

File: cvat-ui/src/actions/review-actions.ts [hunk 2]
```diff
@@ -25,10 +25,14 @@ export enum ReviewActionTypes {
     COMMENT_ISSUE = 'COMMENT_ISSUE',
     COMMENT_ISSUE_SUCCESS = 'COMMENT_ISSUE_SUCCESS',
     COMMENT_ISSUE_FAILED = 'COMMENT_ISSUE_FAILED',
+    REMOVE_ISSUE_SUCCESS = 'REMOVE_ISSUE_SUCCESS',
+    REMOVE_ISSUE_FAILED = 'REMOVE_ISSUE_FAILED',
+    SWITCH_REMOVE_ISSUE_DIALOG = 'SWITCH_REMOVE_ISSUE_DIALOG',
```
Comment (bsekachev) [inline cvat-ui/src/actions/review-actions.ts:30]: Is it used action?

File: cvat-ui/src/actions/review-actions.ts [hunk 3]
```diff
@@ -25,10 +25,14 @@ export enum ReviewActionTypes {
...
     SUBMIT_REVIEW = 'SUBMIT_REVIEW',
     SUBMIT_REVIEW_SUCCESS = 'SUBMIT_REVIEW_SUCCESS',
     SUBMIT_REVIEW_FAILED = 'SUBMIT_REVIEW_FAILED',
     SWITCH_ISSUES_HIDDEN_FLAG = 'SWITCH_ISSUES_HIDDEN_FLAG',
+    SWITCH_ISSUES_HIDDEN_RESOLVED_FLAG = 'SWITCH_ISSUES_HIDDEN_RESOLVED_FLAG',
```
Comment (bsekachev) [inline cvat-ui/src/actions/review-actions.ts:35]:
```suggestion
    SWITCH_RESOLVED_ISSUES_HIDDEN_FLAG = 'SWITCH_RESOLVED_ISSUES_HIDDEN_FLAG',
```

**Review by bsekachev — APPROVED** [review]

**Review by nmanovic — APPROVED** [review]

**Comment by nmanovic** [comment]
@TOsmanov , @aschernov , could you please add documentation for the feature?

**Comment by TOsmanov** [comment]
> @TOsmanov , @aschernov , could you please add documentation for the feature?

@nmanovic , yes, sure.

**Comment by ActiveChooN** [comment]
@dvkruchinin, could you add tests, please?

**Comment by dvkruchinin** [comment]
> @dvkruchinin, could you add tests, please?

Sure.

**Pull request merged**